hazelcast/src/main/java/com/hazelcast/hotrestart/HotRestartBackupService.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.hotrestart;
+
+/**
+ * Service for starting cluster-wide hot restart data backups, determining the local backup state and interrupting a currently
+ * running local hot restart backup.
+ */
+public interface HotRestartBackupService {
```

Comment:
Are these classes part of public API? If not then it's better to move all to an `internal` package?